app/models/hearing_summary.rb

```diff
@@ -39,8 +39,6 @@ def court_centre
     HmctsCommonPlatform::CourtCentre.new(body["courtCentre"])
   end
 
```

Comment:
models/hearing_summary.rb has a resulted? method. Wondering if the logic for calculating this is faulty/outdated, and we could now use the hasSharedResults attribute?